src/app/static/src/app/store/dataExploration/mutations.ts

```diff
@@ -0,0 +1,25 @@
+import {MutationTree} from "vuex";
+import {PayloadWithType} from "../../types";
+import {mutations as languageMutations} from "../language/mutations";
+import {Error} from "../../generated";
+import {DataExplorationState} from "./dataExploration";
+
+export enum DataExplorationMutation {
+    ErrorReportError = "ErrorReportError",
+    ErrorReportSuccess = "ErrorReportSuccess"
+}
+
+export const mutations: MutationTree<DataExplorationState> = {
+
+    [DataExplorationMutation.ErrorReportError](state: DataExplorationState, action: PayloadWithType<Error>) {
```

Comment:
These two mutations are duplicated from root mutations, so probably want pulling out into `errorMutations` as we have done with `languageMutations` below.